funcions/6.test.js

```diff
@@ -0,0 +1,17 @@
+const { add } = require('./6')
+
+describe('add', () => {
+	it('addFive(3) returns 8', () => {
+		const addFive = add(5)
+		const result = addFive(3)
+		const expected = 8
+		expect(result).toEqual(expected)
+	})
+
+	it('First value provided to add function will be a base value for future addition ', () => {
+		const addTem = add(10)
+		const result = addTem(5)
```

Comment:
Just to note, that it could also be written like this:
```suggestion
		const result = add(10)(5)
```